tests/integration_tests/performance/test_boottime.py

```diff
@@ -111,8 +112,13 @@ def test_initrd_boottime(test_microvm_with_initrd, record_property, metrics):
 
 def _test_microvm_boottime(vm, max_time_us=MAX_BOOT_TIME_US):
     """Auxiliary function for asserting the expected boot time."""
-    boot_time_us = 0
-    timestamps = vm.find_log_message(TIMESTAMP_LOG_REGEX)
+    boot_time_us = None
```

Comment:
mh, with this being `None` instead of zero, the assert below needs to be updated (otherwise we get `TypeError: '>' not supported between instances of 'NoneType' and 'int'`)

Reply:
Good point... maybe it should just be 0